support-frontend/assets/helpers/page/analyticsAndConsent.ts

```diff
@@ -9,23 +9,30 @@ import {
 	setReferrerDataInLocalStorage,
 	trackAbTests,
 } from 'helpers/tracking/ophan';
+import { init as initQuantumMetric } from 'helpers/tracking/quantumMetric';
 import { isPostDeployUser } from 'helpers/user/user';
-import * as logger from 'helpers/utilities/logger';
+import { init as initLogger } from 'helpers/utilities/logger';
 import 'helpers/internationalisation/country';
+
 // ----- Functions ----- //
+
 // Sets up GA and logging.
-export function analyticsInitialisation(
+const analyticsInitialisation = (
```

Comment:
Given the discussion elsewhere around a preference for function declarations, should we stick with that approach here?